blocks/job-openings/job-openings.js

```diff
@@ -1,190 +1,11 @@
-import { readBlockConfig } from '../../scripts/lib-franklin.js';
+const tagString = '<iframe src="//www.careers-page.com/netcentric" class="embed-job-openings"></iframe>';
+const range = document.createRange();
+const jobOpeningsBlock = document.querySelectorAll('div.job-openings');
```

Comment:
why querySelectorAll, and also, why not use the decorate function? that fuction already provides the block element to start with